runtime/sunvmi/sunvmi.c

```diff
@@ -1334,9 +1330,8 @@ SunVMI_LifecycleEvent(J9JavaVM* vm, IDATA stage, void* reserved)
 
 	case INTERPRETER_SHUTDOWN:
 	{
-		j9sl_close_shared_library(VM.threadLibrary);
```

Comment:
Where is the replacement for this call?

Reply:
Since the library is not opened a second time, it second instance doesn't need to be closed. I don't know that any of the core libraries we preload are closed. I did zero out the variable in jvminit.c